packages/integration-tests/test/shared/utils.ts

```diff
@@ -0,0 +1,11 @@
+import { BigNumber } from 'ethers'
+
+export const defaultTransactionFactory = () => {
```

Comment:
I feel like this should ideally take each of the possible args as input in an object and then assign those args if they are passed in, otherwise use the default values, will clean up a bunch of code